test/test.js

```diff
@@ -120,7 +122,7 @@ suite('trying client', () => {
   });*/
 
 
-  test('docker exec wc -c', async () => {
+  it('docker exec wc -c', async () => {
```

Comment:
why the switch to bdd? I think maybe what is happening here is new mocha uses [config instead of mocha.opts](https://mochajs.org/#configuring-mocha-nodejs). Copy over mocha.opts to config format and this should go back to normal

Reply:
"Fixed" it